docs/devel/releasing.md

```diff
@@ -117,7 +117,7 @@ oncall.
 Before proceeding to the next step:
 
 ```sh
-export GITHASH=v1.2.0-alpha.2.164+b44c7d79d6c9bb
+export gitcommithash=v1.2.0-alpha.2.164+b44c7d79d6c9bb
```

Comment:
You are right. It was lowercase in the previous versions upper case with underscore is better